src/less/less/ds4/mixins.less

```diff
@@ -107,433 +107,415 @@
     font-weight: @font-weight-regular;
 }
 
-// Font-Icon Base mixins (font-icon is deprecated)
-//--------------------
+// Iconography
+//-----------------------------
 
-.icon-base() {
-    color: @color-icon;
-    display: inline-block;
-    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
-    font-family: @iconfont-name;
-    -moz-osx-font-smoothing: grayscale;
-    -webkit-font-smoothing: antialiased;
-    font-style: normal;
-    font-weight: normal;
-    speak: none;
-    text-transform: none;
+.dimensions(@width; @height) {
+    height: @height;
+    width: @width;
 }
 
-// Font-Icon mixins (font-icon is deprecated)
-//---------------------------------------
-
-.icon-arrowdown-mixin() {
-    content: "\e601";
+// base styles for BEM svg.icon block
+.inline-icon-base(@width: 1rem; @height: 1rem) {
```

Comment:
I'm pretty sure I moved these 3 mixins out of ds4 into common, and I got rid of the dimension mixin (seems superfluous)... so maybe got lost in the merge?